metaserver/src/main/scala/scala/meta/languageserver/WorkspaceCommand.scala

```diff
@@ -0,0 +1,15 @@
+package scala.meta.languageserver
+
+import enumeratum.Enum
+import enumeratum.EnumEntry
+import enumeratum.EnumEntry.Uncapitalised
+
+sealed trait WorkspaceCommand extends EnumEntry with Uncapitalised
```

Comment:
I went for a `lowerCamelCase` convention here and thanks to `Uncapitalised` this is automatically handled by `withNameOption` and `entryName`